cla_public/apps/checker/constants.py

```diff
@@ -132,6 +132,7 @@
     "publiclaw",
     "aap",
     "employment",
+    "traffickingslavery",
```

Comment:
Where does `traffickingslavery` come from?
does it come from the backend?

Reply:
Yeah, this isn't needed. Removed. Thanks.